stories/button/button.html

```diff
@@ -1,5 +1,11 @@
-<button class="fd-button fd-button--success">
-  test123
-</button>
+<script>
+  export let button = {};
+</script>
 
-<button class="fd-button sap-icon--cart"></button>
\ No newline at end of file
+<div dir="{button.dir}">
+  <button
+    class="fd-button
+    {button.icon ? button.icon : ''}
+    {button.emphasized ? 'fd-button--emphasized' : ''}"
+  >{button.text}</button>
+</div> 
```

Comment:
Building out button a little bit to show how we need to do these. When we transition this into docs, each example should be it's own `export const name =...` in the `.stories.js` file. Putting a big glob of html in here won't work well for documentation.

For visual regression tests, we can build this out in the future to have a `button.visual.js` where everything from `button.stories.js` is added to that on one page and that's where the visual image is taken. That's how we've been doing it in react so that the visual regression images don't take forever.